src/samples/simple-freedom-chat/freedom-module.ts

```diff
@@ -68,18 +68,18 @@ function makePeerConnection(name:string) {
   return pc;
 }
 
-var a :PeerConnection<SignallingMessage> = makePeerConnection('A');
-var b :PeerConnection<SignallingMessage> = makePeerConnection('B')
+var a :PeerConnection<signals.Message> = makePeerConnection('A');
+var b :PeerConnection<signals.Message> = makePeerConnection('B')
 
 // Connect the two signalling channels. Normally, these messages would be sent
 // over the internet.
-a.signalForPeerQueue.setSyncHandler((signal:SignallingMessage) => {
-  log.info('a: sending signal to b.');
-  b.handleSignalMessage(signal);
+a.signalForPeerQueue.setSyncHandler((message:signals.Message) => {
+  log.info('a: sending signal message to b.');
+  b.handleSignalMessage(message);
 });
-b.signalForPeerQueue.setSyncHandler((signal:SignallingMessage) => {
-  log.info('b: sending signal to a.');
-  a.handleSignalMessage(signal);
+b.signalForPeerQueue.setSyncHandler((message:signals.Message) => {
+  log.info('b: sending signal message to a.');
```

Comment:
"signal message" reads weird..."signalling message" would be better (or "signal" for short).